src/app/modules/main/wallet_section/send_new/path_model.nim

```diff
@@ -65,6 +65,13 @@ QtObject:
   method rowCount(self: PathModel, index: QModelIndex = nil): int =
     return self.items.len
 
+  proc countChanged(self: PathModel) {.signal.}
```

Comment:
we should not add or use this, we can use model.`ModelCount.count` in cases we want the updated count on the qml side.

Reply:
What if I want it on the Nim side?

Reply:
then lets remove the `{.slot.}` signature from here

Reply:
I used `rowCount`, updated.

Reply:
`rowCount()` should be public by default